pom.xml

```diff
@@ -285,4 +298,188 @@
+        <groupId>io.stargate.auth.table</groupId>
+        <artifactId>auth-table-based-service</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.auth.table</groupId>
+        <artifactId>authentication</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.core</groupId>
+        <artifactId>core</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.cql</groupId>
+        <artifactId>cql</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.graphql</groupId>
+        <artifactId>graphqlapi</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.health</groupId>
+        <artifactId>health-checker</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.db</groupId>
+        <artifactId>persistence-api</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.db.cassandra</groupId>
+        <artifactId>persistence-cassandra-3.11</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.db.cassandra</groupId>
+        <artifactId>persistence-cassandra-4.0</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.db</groupId>
+        <artifactId>persistence-common</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.db.dse</groupId>
+        <artifactId>persistence-dse-6.8</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.db</groupId>
+        <artifactId>persistence-test</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.web</groupId>
+        <artifactId>restapi</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.starter</groupId>
+        <artifactId>stargate-starter</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.stargate.it</groupId>
+        <artifactId>testing</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.dropwizard</groupId>
+        <artifactId>dropwizard-core</artifactId>
+        <version>${dropwizard.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.dropwizard</groupId>
```

Comment:
Should we maybe create an http pom/bom for these sort of things?

Reply:
Yeah also if there's any common code we can factor we could consider extracting an http utility module.

Reply:
https://github.com/stargate/stargate/issues/239